### PR TITLE
LPS-84717 Add Sort to Structured Content

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test

File: modules/apps/headless-apio/structured-content/structured-content-apio-impl/src/main/java/com/liferay/structured/content/apio/internal/provider/SortProvider.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * @author Cristina González
+ */
+@Component
+public class SortProvider implements Provider<Sort> {
+
+	@Override
+	public Sort createContext(HttpServletRequest httpServletRequest) {
+		String sortParam = httpServletRequest.getParameter("sort");
+
+		return new Sort(_sortParser.parse(sortParam));
```
Comment (nhpatt) [inline modules/apps/headless-apio/structured-content/structured-content-apio-impl/src/main/java/com/liferay/structured/content/apio/internal/provider/SortProvider.java]: 👏 👏 👏

**Comment by nhpatt** [comment]
LGTM 👍 

Good job, thanks for the PR :D

**Comment by nhpatt** [comment]
Let's discuss it on monday... I think Sort should be inside apio-architect or at least in headless-apio/portal-api :)

**Pull request merged**